### ISSUE TITLE
Geonode 4 running on Docker

### ISSUE THREAD

**Issue by brunolvs** [GeoNode/geonode#10829]
Does anyone have Geonode 4.xx running on Docker with no problems? All features working, etc.... Can you tell me how you get it? For me is 98%, only Edit Style is not working.

**Comment by Landluemmel** [comment]
Same for me, style editing is not working because of an authentification issue.

**Comment by mwallschlaeger** [comment]
Yes i'm also familiar with this issue about the styling. Related to other issues related to upload, I usually add `PROXY_URL=""` to the .env file.

**Comment by kikislater** [comment]
GEOSERVER_LB_HOST_IP is the most common problem for editing styles.
In my side, what I found is ip or address of this setting should be accessible for remote machine who plays with Django interface

**Comment by giohappy** [comment]
>In my side, what I found is ip or address of this setting should be accessible for remote machine who plays with Django interface

I'm not sure what you mean. The Geoserver location must be the address of the GeoNode's own Geoserver. It can be located to a different address of course. What is important is that the various Geoserver variables are configured correctly and GeoNode can reach it.

**Comment by giohappy** [comment]
Closing as part of a repository cleanup: this issue was last updated before 2025-01-01. Not a judgment on the report itself — feel free to reopen or file a fresh issue against `main` if it's still relevant. Thanks for the report.
